pep-0646.rst

```diff
@@ -446,8 +455,26 @@ instance is *not* allowed:
 
 ``*args`` is the only case where an argument can be annotated as ``*Ts`` directly;
 other arguments should use ``*Ts`` to parameterise something else, e.g. ``Tuple[*Ts]``.
-If ``*args`` itself is annotated as ``Tuple[*Ts]``, the old behaviour still applies:
-all arguments must be a ``Tuple`` parameterised with the same types.
+
+Using a Tuple of Types
+''''''''''''''''''''''
+
+Similarly, ``*args`` can be annotated as a star-prefixed type tuple:
+
+::
+
+    def foo(*args: *Tuple[int, int]): ...
+    # Very similar to:
+    def bar(arg1: int, arg2: int): ...
+    # But a) `foo` may be more convenient if e.g. passing `args` on
+    # to something else, and b) `foo` disallows keyword arguments.
```

Comment:
I would move this back to regular text, with another example. A bit more stretched out vertically, sure, but having this much explanation in a comment seems just wrong.

Reply:
Done.